Katello/katello
# Fixes #34453 - print more info on corrupt/missing content

#### What are the changes introduced in this pull request?

#### Considerations taken when implementing this change?

#### What are the testing steps for this pull request?

1.  spin up a 3.18 and switch to pulp2 (or 6.9)
2.  sync some content, maybe some RH content too with immediate
3.  publish it in a CV
4.  'corrupt' some files in /var/lib/pulp/content/  by either deleting them or echo'ing extra data to them
5. run foreman-rake content prepare
6. run foreman-rake  content migration-stats
examine the resulting file showing the corrupt content, you should see more information now:

```
filename,id,Repository Name,Content View Name,Content View Version
cat-1.0-1.noarch.rpm,2,test,Default Organization View,1.0
cat-1.0-1.noarch.rpm,2,test,Mycv,1.0
tiger-1.0-4.noarch.rpm,7,test,Default Organization View,1.0
tiger-1.0-4.noarch.rpm,7,test,Mycv,1.0
```
